tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/UpgradeAssistantSettingsUtil.java

```diff
@@ -85,6 +85,22 @@ public boolean accept( File dir, String name )
         return null;
     }
 
+    public static <T> T getObjectFromStore( Class<T> clazz, String suffix ) throws IOException
```

Comment:
if we make one project match one json, it will be easier to make "change" for them